python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from dataclasses import dataclass, field
+from itertools import chain
+from typing import Any, Callable, Dict, Optional
+
+from whylogs.core.datatypes import (
+    DataType,
+    Fractional,
+    Integral,
+    StandardTypeMapper,
+    String,
+    TypeMapper,
+)
+from whylogs.core.metrics import StandardMetric
+from whylogs.core.metrics.metrics import (
+    Metric,
+    MetricConfig,
+    OperationResult,
+    register_metric,
+)
+from whylogs.core.metrics.multimetric import MultiMetric, SubmetricSchema
+from whylogs.core.preprocessing import PreprocessedColumn
+
+logger = logging.getLogger(__name__)
+
+
+class DefaultSchema(SubmetricSchema):
+    def resolve(self, name: str, why_type: DataType, fi_disabled: bool = False) -> Dict[str, Metric]:
+        metrics: Dict[str, Metric] = {
+            "counts": StandardMetric.counts.zero(MetricConfig()),
+            "types": StandardMetric.types.zero(MetricConfig()),
+            "cardinality": StandardMetric.cardinality.zero(MetricConfig()),
+        }
+
+        if isinstance(why_type, Integral):
+            metrics["distribution"] = StandardMetric.distribution.zero(MetricConfig())
+            metrics["ints"] = StandardMetric.ints.zero(MetricConfig())
+            if not fi_disabled:
+                metrics["frequent_items"] = StandardMetric.frequent_items.zero(MetricConfig())
+        elif isinstance(why_type, Fractional):
+            metrics["distribution"] = StandardMetric.distribution.zero(MetricConfig())
+        elif isinstance(why_type, String) and not fi_disabled:
```

Comment:
consider this to make it easier to write metrics that cover string values on images.
```suggestion
        if not fi_disabled:
```